### DISCUSSION TITLE
Example of usage as a module from CDN in the browser

### DISCUSSION THREAD

**Discussion by fralik** [josdejong/mathjs#3106] [Q&A] [answered]
I am trying to find a way to load mathjs library as a module in the browser. The documentation is very brief about it and suggests `import { sqrt } from 'mathjs'`. My issue - I do not use node, I have an html and .js file. I want to do something like `import { matrix } from https://unpkg.com/mathjs@12.2.0/lib/esm/number.js`. This doesn't work. Is there a way it could work?

I am also OK with building mathjs locally in order to add it to my html/js files, but I guess I still need to create a single bundle file from which I can import.

Unfortunately, simply using the browser version doesn't work for me neither. I am trying to load mathjs in a web worker alongside another library, which is ESM-only.

I believe the question is similar to https://github.com/josdejong/mathjs/discussions/2784

**Comment by fralik** [comment, marked as answer]
Looks like I found the solution!

`import * as math from 'https://cdn.jsdelivr.net/npm/mathjs@12.2.0/+esm';`

**Reply by josdejong** [reply]
Ahh, nice! Thanks for sharing Vadim.
